firestore/build.gradle.kts

```diff
@@ -31,4 +32,13 @@ dependencies {
     androidTestImplementation(Config.Libs.Test.rules)
     androidTestImplementation(Config.Libs.Test.mockito)
     androidTestImplementation(Config.Libs.Arch.paging)
+    implementation(kotlin("stdlib-jdk7", kotlin_version))
```

Comment:
Whoa there! @samtstern Is FUI accepting Kotlin now? From the [FAQ](https://developer.android.com/kotlin/faq):
![image](https://user-images.githubusercontent.com/9490724/54548744-d1afe180-4965-11e9-9c04-3d0a770b5d1e.png)

Typically, libs will be written _entirely_ in Kotlin (not postfixed like this) and only if it offers some sort of syntactic benefit that isn't available in Java (like coroutines or multi-platform stuff, see https://github.com/square/okio). Here, I have trouble seeing what's worth the 1MB and cognitive overhead of ensuring our APIs stay idiomatic in both Java _and_ Kotlin.

Reply:
Yeah I am pushing the Kotlin question to the end.  If the PR is worth having we can translate it back to Java at the last step.

Reply:
SGTM 👍